handlers/misttriggers/live_tracklist.go

```diff
@@ -70,13 +73,33 @@ func (d *MistCallbackHandlersCollection) TriggerLiveTrackList(w http.ResponseWri
 		return
 	}
 
-	// Start push per each video track
+	// Upload each track (transcoded rendition) returned by Mist to S3
 	for i := range tracks {
+		// Only produce a rendition for each video track, selecting best audio track
 		if tracks[i].Type != "video" {
-			// Only produce an rendition per each video track, selecting best audio track
 			continue
 		}
-		destination := fmt.Sprintf("%s/%s__%dx%d.ts?video=%d&audio=maxbps", info.UploadDir, streamName, tracks[i].Width, tracks[i].Height, tracks[i].Index) //.Id)
```

Comment:
Was the bug that the destination URL we were constructing here was using a .ts file extension, but we want a .m3u8 file extension?

Reply:
Yeah